editor/plugins/sprite_2d_editor_plugin.cpp

```diff
@@ -566,6 +566,8 @@ void Sprite2DEditor::_notification(int p_what) {
 			options->get_popup()->set_item_icon(MENU_OPTION_CONVERT_TO_POLYGON_2D, get_editor_theme_icon(SNAME("Polygon2D")));
 			options->get_popup()->set_item_icon(MENU_OPTION_CREATE_COLLISION_POLY_2D, get_editor_theme_icon(SNAME("CollisionPolygon2D")));
 			options->get_popup()->set_item_icon(MENU_OPTION_CREATE_LIGHT_OCCLUDER_2D, get_editor_theme_icon(SNAME("LightOccluder2D")));
+
+			panner->set_viewport(get_viewport());
```

Comment:
I'd move this to ENTER_TREE and add break. Handling both ENTER_TREE and THEME_CHANGED [is redundant](https://github.com/godotengine/godot/pull/89746).